WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -194,12 +198,14 @@ private async Task<ImmutableArray<AliceClient>> CreateRegisterAndConfirmCoinsAsy
 
 		private BobClient CreateBobClient(RoundState roundState)
 		{
+			var arenaRequestHandler = new WabiSabiHttpApiClient(HttpClientFactory.NewBackendHttpClient(Mode.NewCircuitPerRequest));
+
```

Comment:
This arena request handler can be shared by all clients because bob clients only make single requests and the requests are isolated, but I actually prefer this approach since it's a little bit clearer that they are isolated.

Is there any real reason to prefer one approach over the other? @kiminuo?

Reply:
I believe that this can be shared and it would be probably my approach but I don't see any strong advantage at the moment.

Maybe, it may be suprising for the future me & you in the sense "Why do we it this way when it looks like it can be shared? There must be a reason. Let's rather not change the code as it works." So I would add a comment (as always). :)